evaluation/toolqa/utils.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import json
+import requests
+from functools import partial
+import sys
+import gdown
+import zipfile
+import os
+import re
+import string
+
+def download_data(dir):
+    data_path = os.path.join(dir, "data/external_corpus")
+    if os.path.exists(data_path): return data_path
+    url = "https://drive.google.com/uc?id=1zRbHzPW2x4dDcfmphBWlan8cxUCRNmqk"
```

Comment:
Is it safe to expose this url here? Is this a public resource or your personal one?

Reply:
This is public resource. This is the google drive of the data that the ToolQA benchmark requires users to download. (https://github.com/night-chen/ToolQA?tab=readme-ov-file#external-corpus-download)